play/PlayerRules/CharacterCreation/04_01_Weapons.txt

```diff
@@ -398,11 +398,101 @@ unless otherwise stated.
 
 ===== Level 1 Weapons =====
 
----- 
+---- Matthews Model 2x12	$1029
+Damage, Miss chance:
+	140 damage 3 to miss out to 10m

```

Comment:
140 damage, 3 out to 10m
I think it would be fine just to keep the accuracy phrasing consistent throughout the weapons. Or lose the first "to":
 140 damage, 3 miss out to 10m 
Also, please add a comma after damage.